scene/2d/physics_body_2d.cpp

```diff
@@ -535,12 +535,12 @@ real_t RigidBody2D::get_inertia() const {
 
 void RigidBody2D::set_weight(real_t p_weight) {
 
-	set_mass(p_weight / (real_t(GLOBAL_DEF("physics/2d/default_gravity", 98)) / 10));
+	set_mass(p_weight / (real_t(GLOBAL_DEF("physics/2d/default_gravity", 980)) / 10));
```

Comment:
Any reason why this have to be divided by `10` in the first place? :eyes:

Reply:
I don't know, the whole weight/mass system feels messy to me. We should probably have just one variable to control an object's effective gravity, rather than two (one of them just setting a "computed value" of sorts).